### DISCUSSION TITLE
[Feature] Parse markdown in List View descriptions

### DISCUSSION THREAD

**Discussion by rogercreagh** [TechbeeAT/jtxBoard#449] [Ideas]
Currently if markdown codes are present in the first 6 lines of the description they are displayed as plain text in the List View. This is ugly.

Would it be possible to simply always parse the truncated description for markdown (probably before truncating it so you don't end up with unclosed tags).

If it is not simple, or is too much overhead, to always parse markdown then the use of FMTTYPE element parameter as per #446 above would tell you what to do.

**Comment by patrickunterwegs** [comment]
Hey, are you talking about the list or the detail view?
The list view currently doesn't interpret markdown at all! 
But do you also have a problem in the detail view? Maybe you can also add a screenshot (if it doesn't contain any sensitive data)

**Comment by rogercreagh** [comment]
> Hey, are you talking about the list or the detail view? The list view currently doesn't interpret markdown at all! But do you also have a problem in the detail view? Maybe you can also add a screenshot (if it doesn't contain any sensitive data)

List View as opposed to Grid View or Kanban View or Compact view on the main 3dots menu. The other views only show the first few chars of the description which is more or less ok but would be improved if there happens to be a markdown code in the first few chars. 
For these views (and possibly Detail View as well, it might be better to strip markdown tags but I'm not sure there's an easy way to do that. Stripping Header tags `###` etc in the displayed list view descriptions might be a good idea though as if you desplay them as parsed markdown they'll screw up the screen layout.

Assuming by detail view you mean the details screen as opposed to the list format view options, then no, the only problem there is that if you turn markdown off (because you wanted it off somewhere else and forgot to turn it back on) then the codes show. If you used FMTTYPE then you wouldn't have to reply on the setting.

**Comment by patrickunterwegs** [comment]
I thought about it, I like neither the idea of stripping tags, nor the idea of markdown formatting in the list.
- stripping tags: this would mean that you don't see what's actually there without knowing why, so I'd rather keep the tags and be consistent
- use the actual formatting: this could be very ugly in the layout if people use long headers. It's better to keep the list consistent 

We can move this to discussions, but for now I don't see a strong reason to change the current behavior
